serverless/serverless-architecture.adoc

```diff
@@ -6,3 +6,6 @@ include::modules/serverless-document-attributes.adoc[]
 
 include::modules/knative-serving.adoc[leveloffset=+1]
 include::modules/knative-cli.adoc[leveloffset=+1]
+
+== Knative Eventing
+A developer preview version of Knative Eventing is available for use with {ServerlessProductName}, however this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information about Knative Eventing, including installation instructions and samples, see the https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
```

Comment:
We do not document Developer Preview features, only those in Technology Preview. I would double check if this should be included yet.